src/components/Tutorial/Tutorial2StepContext2Subthought.native.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { Fragment } from 'react'
+import { store } from '../../store'
+
+import {
+  TUTORIAL_CONTEXT,
+  TUTORIAL_CONTEXT2_PARENT,
+  TUTORIAL_VERSION_BOOK,
+  TUTORIAL_VERSION_JOURNAL,
+  TUTORIAL_VERSION_TODO,
+} from '../../constants'
+
+import { head, headValue, isRoot, joinConjunction } from '../../util'
+
+// selectors
+import { getContexts, getChildrenRanked } from '../../selectors'
+
+import TutorialHint from './TutorialHint'
+import StaticSuperscript from '../StaticSuperscript'
+
+import { Child, Path, ThoughtContext } from '../../@types'
+import { Text } from '../Text.native'
+import { commonStyles } from '../../style/commonStyles'
+
+type TutorialChoice = typeof TUTORIAL_CONTEXT2_PARENT
+
+interface IComponentProps {
+  rootChildren: Child[]
+  tutorialChoice: keyof TutorialChoice
+  cursor?: Path
+}
+
+const { smallText, italic } = commonStyles
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+const context2SubthoughtCreated = ({ rootChildren, tutorialChoice }: IComponentProps) => {
+  const state = store.getState()
+  // e.g. Work
+  return (
+    rootChildren.find(child => child.value.toLowerCase() === TUTORIAL_CONTEXT2_PARENT[tutorialChoice].toLowerCase()) &&
```

Comment:
theres opportunity for a single util function which compares two values toLowerCase